R/S3_summary.R

```diff
@@ -102,7 +102,8 @@ summary.horvitz_thompson <- function(object, ...) {
 summarize_tidy <- function(object, test = "t", ...) {
   remove_cols <- c("term", "outcome")
 
-  # This is ugly SO THAT summary(fit)$coefficients returns something like lm does.
+  # This is ugly SO THAT summary(fit)$coefficients returns
+  # something like lm does.
```

Comment:
Edit the comment instead of wrapping it, eg

```
  # Ugly so that summary(fit)$coefficients matches lm() 
```

Reply:
will fix